src/components/flat-table/flat-table-cell/flat-table-cell.component.d.ts

```diff
@@ -4,6 +4,10 @@ export interface FlatTableCellProps {
   /** Content alignment */
   align: string;
   children: React.ReactNode | string;
+  /** Number of columns that a cell should span */
```

Comment:
Should these not have `?` as they aren't required?

Reply:
you got me there, fixed